pkg/kudoctl/cmd/operator_verify.go

```diff
@@ -0,0 +1,38 @@
+package cmd
```

Comment:
let's many separate this into a different PR?

Reply:
:+1: